### ISSUE TITLE
Never manage to install it inside VirtualBox

### ISSUE THREAD

**Issue by c02y** [archlinux/archinstall#788]
I tried to install ArchLinux 2021.10.01 iso and 2021.12.01 iso after they were released using archinstall script, no matter how many times I tried, never manage to install it. It crashed in the middle of the process or crashed in the end. 

I assume this script is not tested inside VirtualBox. If this assumption is false, I can provide more info about the install process and crash message if you need it.

FYI:
Host: ArchLinux or Manjaro
VM: VirtualBox

**Comment by dylanmtaylor** [comment]
I've successfully installed it in VirtualBox before.  Including on the latest release which should be on the December ISO.  Could you please provide the actual error so that we can look into it?

**Comment by Torxed** [comment]
The `/var/log/archinstall.log` would be very helpful in diagnosing what went wrong and why :) Even if it failed.

**Comment by c02y** [comment]
Crash1:
1. 0: BlockDevice(/run/archiso/bootmnt/xxx) **or** 1: BlockDevice(/dev/sda. size=18GBxxx)
2. 0: Wide all selected drvices and use
3. 1: ext4
crashed
![2021-12-07_03-52_1](https://user-images.githubusercontent.com/2135996/144913483-26828613-80b7-4d26-b931-b847152a7e3b.png)

Crash2:
1. 2: BlockDevice(None, size=0.8GB)
2. 0: Wide all selected drvices and use
3. 1: ext4
4. 0: desktop
5. 6: i3
6: 1: i3-wm
7. 5: VMware / VirtualBox (open-source)
8. 0: linux
9. a few simple options 
crashed
![2021-12-07_04-06](https://user-images.githubusercontent.com/2135996/144914941-8eb122a2-f1b6-4d7e-8b21-7de6f2711cba.png)

The final part of `/var/log/archinstall/install.log` for Crash2 is:
![2021-12-07_04-10](https://user-images.githubusercontent.com/2135996/144915392-61c3a073-d6a7-468a-9799-8917e0dce595.png)


I can try more, such as choose different filesystem, different kernel.

**Comment by c02y** [comment]
`live CD to format` and `format anything other than /run/archiso/bootmnt/xxx or /dev/sr0`

Sorry, I've installed dozens of Linux distros in VirtualBox and on real hardware, never seeing the above concepts.

**Comment by c02y** [comment]
I managed to get the download packages process following https://www.youtube.com/watch?v=8mC5IHh-dj8

The only differences I chagned:
1. Select on network interface to configure: choose `1: Use NetworkManager` instead of `2: enp0s3`
2. timezone: specify the timezone instead of `UTC`

yes, it didn't crash like Crash1, so it just randomly crashes?


**UPDATE**:
I tried the exact Crash1 and Crash2 processes multiple times, and they sometimes crash, sometimes successfully install Archlinux. So I don't know what is going wrong. I'll close this for now.
